mapboxgl/viz.py

```diff
@@ -268,3 +268,39 @@ def add_unique_template_variables(self, options):
             clusterRadius=self.clusterRadius,
             clusterMaxZoom=self.clusterMaxZoom
         ))
+
+
+class ChloroplethViz(MapViz):
+    """Create a heatmap viz"""
+
+    def __init__(self,
+                 data,
+                 color_property=None,
+                 color_stops=None,
+                 color_default='grey',
```

Comment:
We should add support for `outline-color` and `outline-width` properties here to control the style of the border around the choropleth.  

I recommend handing these options by creating a second `choropleth-line` layer, using the line type in Mapbox GL style spec, where the `line-color` and `line-width` can be controlled easily.

Reply:
Addressed with 0cd6da8.